src/gui/TextEditor.cpp

```diff
@@ -904,8 +905,66 @@ auto TextEditor::blinkCallback(TextEditor* te) -> gint {
     return false;
 }
 
+void TextEditor::setTextToPangoLayout(PangoLayout* pl) const {
```

Comment:
Note: Although the original code is undocumented, I think this would be a good opportunity to add some comments clarifying the intention. This PR is targeting release-1.1, but we'll be merging into master at some point anyways.